packages/styled-components/src/secretInternals.js

```diff
@@ -1,8 +1,10 @@
 // @flow
+/* eslint-disable */
 
-import StyleSheet from './models/StyleSheet';
+import StyleSheet from './sheet';
+import { masterSheet } from './models/StyleSheetManager';
 
-/* eslint-disable */
 export const __DO_NOT_USE_OR_YOU_WILL_BE_HAUNTED_BY_SPOOKY_GHOSTS = {
```

Comment:
Can we just make this `__INTERNAL__`?

Reply:
Let's leave that for a future PR/discussion?